src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/IsNullOrEmptyConverter.shared.cs

```diff
@@ -10,6 +11,11 @@ namespace Xamarin.CommunityToolkit.Converters
 	/// </summary>
 	public class IsNullOrEmptyConverter : ValueConverterExtension, IValueConverter
 	{
+		/// <summary>
+		/// A value that specifies whether or not invert the result.
+		/// </summary>
+		public bool InvertCheck { get; set; }
```

Comment:
I think better name would be simply `Invert` or `InvertResult`

Reply:
Done!